specs/phase1/shard-transition.md

```diff
@@ -280,7 +280,10 @@ def get_shard_transition(beacon_state: BeaconState,
         if proposal.signature != BLSSignature():
             proposer_signatures.append(proposal.signature)
 
-    proposer_signature_aggregate = bls.Aggregate(proposer_signatures)
+    if len(proposer_signatures) > 0:
+        proposer_signature_aggregate = bls.Aggregate(proposer_signatures)
+    else:
+        proposer_signature_aggregate = BLSSignature()
```

Comment:
Do we define the empty signature somewhere? Is that 0xc000...

Reply:
The python here would mean all zeroes (every ssz type is all zeroes by default). But agree that this is way to vague and can be confused with an empty signature.

Reply:
I remember we had `EMPTY_SIGNATURE` constant in 2019. I would be happy to add it back.

Reply:
We can have, `NO_SIGNATURE`(0x0000...) and `EMPTY_SIGNATURE` (0xc000...) for clarity

Reply:
I've been treating `EMPTY_SIGNATURE` as (0x0000...) as I believe that's what we called in it the spec in 2019. I'm happy to do a re-name and make
* `NO_SIGNATURE` (0x0000...) (always verifies false)
* `EMPTY_SIGNATURE` (0xc000...) 

However, we have to be careful here as `AggregatePublicKeys([])` will always verify false according to the BLS Standard.

So `EMPTY_SIGNATURE` will only verify true against a list of `EMPTY_PUBLIC_KEYS`. We must have `1..*` `EMPTY_PUBLIC_KEYS` verifying true. (Note: or some other combination of public keys that add to 0 mod r).

Reply:
Or `EMPTY_SIGNATURE`(0x0000) and `ZERO_SIGNATURE` (0xc000) I'm not married to any name.

Reply:
Hey I'm redirecting this discussion back to the original #1713 thread: https://github.com/ethereum/eth2.0-specs/issues/1713#issuecomment-628518937 😄